client/src/main/java/org/evosuite/Properties.java

```diff
@@ -1293,7 +1293,7 @@
 	public static boolean ERROR_BRANCHES = false;
 
 	public enum ErrorInstrumentation {
-		ARRAY, ARRAYLIST, CAST, DEQUE, DIVISIONBYZERO, LINKEDHASHSET, LINKEDLIST, NPE, OVERFLOW, QUEUE, STACK, VECTOR
+		ARRAY, ARRAYLIST, CAST, DEQUE, DIVISIONBYZERO, LINKEDHASHSET, LINKEDLIST, NPE, OVERFLOW, QUEUE, STACK, VECTOR, LIST
```

Comment:
Are ARRAYLIST, LINKEDLIST, etc. redundant and covered by LIST / DEQUE? If so, can't we get rid of the redundant types of instrumentation?